corehq/ex-submodules/couchexport/models.py

```diff
@@ -404,7 +404,7 @@ def get_items_in_order(self, row):
                 i = self.row_positions_by_index[column.index]
                 val = row_data[i]
             except KeyError:
-                val = SCALAR_NEVER_WAS
+                val = scalar_never_was
 
             if issubclass(type(column), ComplexExportColumn):
                 for value in column.get_data(val):
```

Comment:
For SplitColumns the logic is here: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/couchexport/models.py#L294-L316